src/components/Card.tsx

```diff
@@ -261,6 +280,15 @@ const CardTop = (props: {
                 </View>
                 <RegularText style={styles.location}>{printableTime}{hostnameText}</RegularText>
             </View>
+            <TouchableView
+                style={{
+                    paddingRight: 10,
```

Comment:
Some paddingTop is missing here.

Are it would be better to add this to the styles block instead of inlining according to our conventions.

Reply:
The paddingTop is still missing :)